frontend-react/src/styles/_components.scss

```diff
@@ -11,7 +11,7 @@ hr {
     @include u-margin-top(4);
 
     &.rs-hr--alternate {
-        @include u-border("2px", $theme-hr-alternate-border-color);
+        @include u-border("2px", "blue-warm-70v");
```

Comment:
If the desired color for that variable has changed, then the variable itself should be changed. If not, then a new style rule needs to be made because this is going to affect other pages.